src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,75 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private int[] supply = new int[10];
+    private int sumSupply = 0;
+    private int counterSupply = 0;
+    private int[] buy = new int[10];
+    private int sumBuy = 0;
+    private int counterBuy = 0;
+    private StringBuilder stringBuilder = new StringBuilder();
```

Comment:
let's not create non-constant variables on class level now - only on method level